### DISCUSSION TITLE
Suggestion: Pest Control Minigame Plugin

### DISCUSSION THREAD

**Discussion by TomMartow** [runelite/runelite#15033] [Ideas]
I have a few suggestions to add to the Pest Control plugin.

## The Concept:
![image](https://user-images.githubusercontent.com/10462926/38825176-88c0ebd8-41a3-11e8-9111-bf421b01cc73.png)

### Remove Default Interface
If enabled, the default interface would be removed, showing no information apart from the custom inactivity percentage, that would be placed below the Knights health and your Damage Points.

### Show Portals on Minimap
If enabled, the Portals would display on the minimap and when out of range the portals would stick to the end of the minimap and would show the direction of the portal regardless of orientation. When the portal is attackable, the portal will change sprite and display a yellow around the edges, similar to the East (Blue) portal in the concept.

###  Show Portal Weaknesses
If enabled, above the portal's would display it's weaknesses. 

![image](https://user-images.githubusercontent.com/10462926/38825267-dea761ee-41a3-11e8-9336-2a7fba3a42ad.png)

**Comment by yani** [comment]
Can I get this assigned? I'm revamping the whole Pest Control plugin. But I'm doing it a bit different.

- Default interface will be kept
- HintArrow pointing to attackable portals
- Portal Weakness will be added (will look at the Slayer plugin?)

I think a hint arrow pointing to the active portal is better than showing them on the minimap. And somebody on discord said I shouldn't remove the interfaces.

**Comment by yani** [comment]
I actually made this quite some time ago: PR https://github.com/runelite/runelite/pull/8096
